governance/polls/Replace Dai oracle with one that is hardcoded to $1 USD - January 8, 2024.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Replace Dai oracle with one that is hardcoded to $1 USD - January 08, 2024
+summary: Signal your support or opposition to replacing the current Dai oracle with one that is hardcoded to $1 USD
+discussion_link: https://forum.makerdao.com/t/spark-spell-proposed-changes/23298
+parameters:
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2024-01-08T16:00:00
+end_date: 2024-01-11T16:00:00
+---
+# Poll: Replace Dai oracle with one that is hardcoded to $1 USD - January 8, 2024
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of Spark Protocol and the Stability Facilitators. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday,January 8 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the replacement of the current Dai oracle on Ethereum with one that is hardcoded to $1 USD to prevent excessive borrows during temporary depeg events.
```

Comment:
```suggestion
The community can vote in this poll to express support or opposition to the replacement of the current Dai oracle on Ethereum with one that is hardcoded to 1 USD to prevent excessive borrows during temporary depeg events.
```

Reply:
Here you could mention that DAI is not collateral on Spark, since if this were the case hardcoding the Oracle could create an attack vector. But it is good as is